docs/run_against_emulator.md

```diff
@@ -0,0 +1,123 @@
+# How to Setup & Run proxy against Cloud Spanner Emulator
+
+## Start a local running Emulator database
+
+0. Export environment variable for local Emulator host:
+    ```
+    export SPANNER_EMULATOR_HOST=localhost:9010
+    ```
+
+1. Verify both 9010 (gRPC server) and 9020 (REST server) are not in used:
+    ```
+    sudo fuser -k 9010/tcp
+    sudo fuser -k 9020/tcp
+    ```
+2. Start a local running Emulator via docker:
+    ```
+    docker pull gcr.io/cloud-spanner-emulator/emulator
+    docker run -p 9010:9010 -p 9020:9020 gcr.io/cloud-spanner-emulator/emulator
+    ```
+    For other ways to start running Emulator, see [Quickstart](https://github.com/GoogleCloudPlatform/cloud-spanner-emulator?tab=readme-ov-file#quickstart) section from [Cloud Spanner Emulator](https://github.com/GoogleCloudPlatform/cloud-spanner-emulator) repo.
+
+3. If not done already, create a gcloud configuration for easier future usage:
+    ```
+    gcloud config configurations create emulator
+    gcloud config set auth/disable_credentials true
+    gcloud config set account emulator-account
+    gcloud config set project emulator-project
+    gcloud config set api_endpoint_overrides/spanner http://localhost:9020/ 
+    gcloud config set instance test-instance
```

Comment:
we need to activate emulator configurations as well or that is optional?

Reply:
It's by default activated if we start with `gcloud config configurations create emulator`